gtest_parallel.py

```diff
@@ -703,16 +722,17 @@ def main():
   assert len(unique_binaries) == len(binaries), (
       "All test binaries must have an unique basename.")
 
-  # Remove files from old test runs.
-  if os.path.isdir(options.output_dir):
-    shutil.rmtree(options.output_dir)
-  # Create directory for test log output.
-  try:
-    os.makedirs(options.output_dir)
-  except OSError as e:
-    # Ignore errors if this directory already exists.
-    if e.errno != errno.EEXIST or not os.path.isdir(options.output_dir):
-      raise e
+  if options.output_dir:
+    # Remove files from old test runs.
+    if os.path.isdir(options.output_dir):
+      shutil.rmtree(options.output_dir)
+    # Create directory for test log output.
+    try:
+      os.makedirs(options.output_dir)
```

Comment:
Here it attempts to create `output_dir` but from a quick look this code can never be reached.

Reply:
It's reachable if the original `output_dir` exists. At this point output_dir should be (original_output_dir)/gtest-parallel-logs. This should probably have a comment. Can you fix the missing base directory on your end?